zict/tests/test_lru.py

```diff
@@ -283,7 +281,7 @@ def test_getitem_is_threasafe():
 
     def f(_):
         barrier.wait()
-        for _ in range(5_000_000):
+        for _ in range(500_000):
             assert lru["x"] == 1
```

Comment:
This test has become an order of magnitude slower.
I'm not too concerned - this is just during continuous collisions from 2+ threads; if only a thread is involved, the overhead of the RLock is nanoseconds.